datamonster_api/lib/aggregation.py

```diff
@@ -4,7 +4,7 @@
 from .errors import DataMonsterError
 
 
-aggregation_periods = {"none", "week", "month", "quarter", "fiscalQuarter", "year"}
+aggregation_periods = {"week", "month", "quarter", "fiscalQuarter", "year"}
```

Comment:
I like this change, but it could be breaking for some users' saved scripts. Have you confirmed that this is what we want?

Reply:
elizabeth is aware of the change, and is comfortable with us making it

Reply:
cool cool cool